docs/data/toolpad/getting-started/why-toolpad.md

```diff
@@ -0,0 +1,98 @@
+# Why Toolpad?
+
+<p class="description">Explains our motivation for building Toolpad, where it differs from other tools and how it could be useful to you.</p>
+
+## Introduction
+
+Toolpad is a developer tool primarily for the admin applications use case. Building an admin panel (aka internal tools) can be intimidating because:
+
+- it is not focused on frontend
+- it is difficult to keep up with the fast evolving landscape of frameworks, libraries and best practices
+- lots of glue code is required to display the data
+- it requires connecting to multiple data sources
+- security of the underlying sensitive data needs to ensured at all times
```

Comment:
This isn't exactly specific to internal applications, personally I'd leave it out

```suggestion
```